sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -362,6 +363,9 @@ static void _print_reply(_ping_data_t *data, gnrc_pktsnip_t *icmpv6,
         }
         printf("%u bytes from %s: icmp_seq=%u ttl=%u", (unsigned)icmpv6->size,
                from_str, recv_seq, hoplimit);
+        if (netif_hdr && netif_hdr->rssi) {
+            printf(" rssi=%d dBm", netif_hdr->rssi);
```

Comment:
[`gnrc_netif_hdr_t::rssi`](api.riot-os.org/structgnrc__netif__hdr__t.html) is of type `int16_t`, so the format string should be `" rssi=%" PRId16 " dBm"`.

Reply:
Ok